solutions/web3-sessions/pages/index.tsx

```diff
@@ -39,6 +42,16 @@ function Home() {
 
       <section className="flex flex-col space-y-4 gap-6">
         <Text variant="h1">Web3 Session with NextAuth.js</Text>
+        {!metamaskInstalled && (
+          <Text>
+            {' '}
+            Please install{' '}
+            <Link href="https://metamask.io/" target="_blank">
+              Metamask
+            </Link>{' '}
+            to use this example.
+          </Text>
+        )}
```

Comment:
I don't think we have to warn fro this early on, having the warning at the end should be enough, it's only there to cover the edge case of someone trying to login ignoring that the example itself mentions the use of Metamask.